app.rb

```diff
@@ -0,0 +1,10 @@
+letters = [
+  "A: alpha", "B: bravo", "C: charlie", "D: delta", "E: echo", 
+  "F: foxtrot", "G: golf", "H: hotel", "I: india", "juliet", 
+  "kilo", "lima", "mike", "november", "oscar", 
+  "papa", "quebec", "romeo", "sierra", "tango",
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.